test/api/Animated.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import View from '../../src/components/View.js';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import Animated from '../../src/api/Animated';
+
+describe.only('Animated.View', () => {
+  before(() => {
+    const jsdom = require('jsdom').jsdom;
+    global.document = jsdom('');
+    global.window = document.defaultView;
+    Object.keys(document.defaultView).forEach((property) => {
+      if (typeof global[property] === 'undefined') {
+        global[property] = document.defaultView[property];
+      }
+    });
+
+    global.navigator = {
+      userAgent: 'node.js',
+    };
+  });
```

Comment:
Would it be quick to pull this out into a global test setup file like we have in our React Native app? We may not run into issues, but it can be a frustrating experience when an individual spec file modifies global state and it's hard to figure out which one.